app/views/shared/_cancel.html.slim

```diff
@@ -0,0 +1,3 @@
+.mt2
+  hr
```

Comment:
Small nitpick, for consistency I don't think we are using `hr` elsewhere, instead we have been using `pt1 border-top` on the containing div.